src/test/java/seedu/address/logic/commands/EditPropertyCommandTest.java

```diff
@@ -40,7 +40,7 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() {
         EditPropertyDescriptor descriptor = new EditPropertyDescriptorBuilder(editedProperty).build();
         EditPropertyCommand editPropertyCommand = new EditPropertyCommand(INDEX_FIRST_PROPERTY, descriptor);
 
-        String expectedMessage = String.format(editPropertyCommand.MESSAGE_SUCCESS, editedProperty);
+        String expectedMessage = String.format(EditPropertyCommand.MESSAGE_SUCCESS, editedProperty);
 
         Model expectedModel = new ModelManager(new PropertyBook(model.getPropertyBook()), new UserPrefs());
```

Comment:
Do use the constructor with both property book and appointment book as the other constructors should be removed asap